walletlibrary/src/main/java/com/microsoft/walletlibrary/wrapper/VerifiedIdCompletionCallBack.kt

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.walletlibrary.wrapper
+
+import com.microsoft.did.sdk.VerifiableCredentialSdk
+import com.microsoft.did.sdk.credential.service.models.issuancecallback.IssuanceCompletionResponse
+import com.microsoft.did.sdk.util.controlflow.Result
+import com.microsoft.walletlibrary.util.VerifiedIdCompletionCallbackException
+
+/**
+ * Wrapper class to wrap the send issuance completion callback to VC SDK.
+ */
+object VerifiedIdCompletionCallBack {
+
+    // sends the issuance callback to VC SDK and returns nothing if successful.
+    internal suspend fun sendIssuanceCompletionResponse(
+        issuanceCompletionResponse: IssuanceCompletionResponse,
+        redirectUrl: String
+    ) {
+        val callbackResult = VerifiableCredentialSdk.issuanceService.sendCompletionResponse(
+            issuanceCompletionResponse,
+            redirectUrl
+        )
+        if (callbackResult is Result.Failure) {
+            throw VerifiedIdCompletionCallbackException(
```

Comment:
nit: VerifiedIdIdIssuanceCompletionCallbackException or something along those lines? so we know it is related to Issuance?